templates/common/.github/workflows/terraform/azure-dev.yml

```diff
@@ -19,9 +19,16 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Log in with Azure
-        uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS }}
+        run: |
+          $info = $Env:AZURE_CREDENTIALS | ConvertFrom-Json -AsHashtable;
+
+          azd login `
+            --client-id "$($info.clientId)" `
+            --client-secret "$($info.clientSecret)" `
```

Comment:
@ellismg Is this properly detected as a secret and masked in the pipeline? I think we've made `AZURE_CREDENTIALS` a secret, but if we use the JSON contents of the secret, I'm not sure if it would still be masked.

Reply:
Good call out here - I don't think that the commands as run get logged - but I will add something to the inline split so we spit out that clientSecret is a secret and gets masked.

Reply:
Added a line here to use `Write-Host` to send the `add-mask` message to the GitHub runner to blind the client secret.